dashdblocal_notebooks/livy-notebooks/README.md

```diff
@@ -0,0 +1,69 @@
+# Configure Livy 
+
+Here we explain how to configure a livy connection so you are able to :
+* work on a Jupyter notebook which you open locally 
+* let it run on a Spark cluster with IBM Db2 Warehouse.
+
+Livy is an open-source REST service for Apache Spark. You can use it with sparkmagic in a Jupyter notebook to execute Spark jobs easily. Read more on the official Apache Livy page : https://livy.apache.org/.
+
+Sparkmagic (https://github.com/jupyter-incubator/sparkmagic) enables to work with remote Spark clusters through Livy in Jupyter notebooks. It will run a Spark job through Livy when you write Spark code on your local Jupyter client. This means you can run a Spark job from your own Jupyter notebook, which is running on your local host.
+
+Let's open a terminal window and execute the following commands:
+> pip install sparkmagic
+
+> jupyter nbextension enable --py --sys-prefix widgetsnbextension
+
+Then retrieve installation directory of sparkmagic with
+> pip show sparkmagic
+
+Move to this directory to install the notebook kernels for Scala, R and Python with:
+> cd your_install_dir
+
+> jupyter-kernelspec install sparkmagic/kernels/sparkkernel
+
+> jupyter-kernelspec install sparkmagic/kernels/sparkrkernel
+
+> jupyter-kernelspec install sparkmagic/kernels/pysparkkernel
+
+Create the config.json file for the sparkmagic configuration file in your home directory with
+> cd 
+
+> mkdir .sparkmagic
+
+> cd .sparkmagic
+
+> wget https://raw.githubusercontent.com/jupyter-incubator/sparkmagic/master/sparkmagic/example_config.json
+
+Create the config.json file as a copy of example_config.json
+> cp example_config.json config.json
+
+Edit the config.json file and specify username, password and url of the IAAS system in the „credentials”-sections of the 3 kernels. "8998" is the default Livy server port. As value for the “auth” attribute enter “Basic_Access”
```

Comment:
IMHO it doesn't make sense to mention IIAS here, without ever expanding the acronym and never mention it ever again. I suggest to just use `Db2 Warehouse` everywhere.